net-analyzer/wireshark/wireshark-4.0.11.ebuild

```diff
@@ -195,8 +195,6 @@ src_configure() {
 		append-cxxflags -fPIC -DPIC
 	fi
 
-	! use lto && filter-lto
```

Comment:
please change this to just filter-lto for now as there's a p.u.semask for wireshark[lto]

Reply:
Good catch, thanks!

Reply:
Did you forget to push that bit or just got it locally?

Reply:
Sorry, branch mismatch when pushing.